src/cephadm/box/setup_ssh.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# SSH
+if [[ ! -f "/root/.ssh/id_rsa" ]]; then
+	mkdir -p ~/.ssh
+	chmod 700 ~/.ssh
+	ssh-keygen -b 2048 -t rsa -f ~/.ssh/id_rsa -q -N ""
+fi
+
+cat ~/.ssh/id_rsa.pub >> /root/.ssh/authorized_keys
+if [[ ! -f "/root/.ssh/known_hosts" ]]; then
+	ssh-keygen -A
+fi
+dnf install -y openssh-server
```

Comment:
@zdover23 had this issue as well. Can you add it to 

https://github.com/ceph/ceph/blob/2612268e0d5e5f33efed56281b423c4459fe7e52/ceph.spec.in#L461-L471

?

Reply:
sure